.build/analyzers/test.step

```diff
@@ -35,6 +35,11 @@
     <call target="set_normal_failure_mode" />
   </target>
 
+  <target name="full">
+    <nant buildfile="${dirs.current.file}${path.separator}nunit.test.step" target="run_full_tests" inheritall="true" if="${test.framework == 'nunit'}" />
```

Comment:
Is there a reason that you didn't add the other buildfile's here?  Granted, they are unlikely to be used, but just curious...

Reply:
I just didn't want to try to implement `run_full_tests` for the files for the other test framework.

Reply:
Understood.  Just wanted to check.  I think that this is fine.